src/physfs_tar.h

```diff
@@ -0,0 +1,299 @@
+#ifndef _INCLUDE_PHYSFS_TAR_H_
+#define _INCLUDE_PHYSFS_TAR_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+#include <ctype.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <time.h>
+
+#ifdef __GNUC__
+#if defined(__MINGW32__) || defined(__MINGW64__)
```

Comment:
MinGW always defines `_WIN32`, so this block shouldn't be needed. `io.h` alone should be sufficient.

However, if this is here only to support the mode functions: see below..